lib/main.coffee

```diff
@@ -93,35 +90,15 @@ module.exports =
 
   # Public: Creates AutocompleteManager instances for all active and future editors (soon, just a single AutocompleteManager)
   activate: ->
-    @getAutocompleteManager()
-    # @activateTimeout = setTimeout(@getAutocompleteManager, 0)
+    run = =>
+      @getAutocompleteManager()
+    setTimeout(run.bind(this), 0)
```

Comment:
@benogle mostly we want to get the package off of timecop's radar. Load is now sorted, going async on activation is the only way I can think of to move it off the list. Most of the time is spent in compilation, although perhaps deferring activation of FuzzyProvider would help somewhat.